### ISSUE TITLE
Consider adding onChage method with button values

### ISSUE THREAD

**Issue by vasilich6107** [Frezyx/group_button#56]
It would be great to have the api that allows setting selected buttons not by index but with actual buttons values.
For example while using in forms

Also it would be great to have `onChange` method with selected button values

**Comment by Frezyx** [comment]
I'm going to make a GroupButton with possibility of generic value types in 5.0.0 version
The idea is very good! I also need this functionality in my personal projects.

**Comment by Frezyx** [comment]
Hello @vasilich6107 !
This feature is available in new 5.0.0 package [release](https://github.com/Frezyx/group_button/releases/tag/5.0.0)

**Comment by vasilich6107** [comment]
@Frezyx this is not what was requested.
The `onSelected` method gives only the current selected value.
What is required - `onChange` method with all selected values.

It is not comfortable to operate with indexes.
While using the `onSelected` method user have to get values by indexes, he also have to handle index deletions rather than simply using the control.

**Comment by VIP-Dev-Invisible** [comment]
> @Frezyx this is not what was requested. The `onSelected` method gives only the current selected value. What is required - `onChange` method with all selected values.
> 
> It is not comfortable to operate with indexes. While using the `onSelected` method user have to get values by indexes, he also have to handle index deletions rather than simply using the control.

I agree too. and  @Frezyx please consider adding an option to choose regular old buttons
